README.md

```diff
@@ -24,9 +24,9 @@ $ LOGXI=* ./nanny
 ```
 Call it via curl:
 ```bash
-curl http://localhost:8080/api/v1/signal --data '{ "name": "my awesome program", "notifier": "stderr", "next_signal": "5s" }'
+curl http://localhost:8080/api/v1/signal --data '{ "name": "my awesome program", "notifier": "stderr", "next_signal": "5s", "all_clear": false }'
```

Comment:
I'm wondering (just a thought, nothing wrong with this approach), wouldn't it be better to have the `all_clear` as separate URL and handler? 

I think that would depend on how exactly would you trigger it, compared to regular signal?

Do you have some workflow that would use this? I'm curious as how the program sending signal would decide if it should signal normally or with `all_clear`?

Reply:
To me it feels more intuitive to set the "all_clear" option via the data in the request (in the same way as e.g. the "notifier").

One use case that I have is a server that pings nanny every minute using a cronjob. When there is a power outage nanny would send an alert after 1m. If the power is restored nanny would send an all-clear.

I haven't yet thought about using the all-clear depending on the state of the program.